cal/cal/models.py

```diff
@@ -583,21 +583,25 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-
-    def queryWeek(self, calendar_ids=None, start=None, end=None):
-        queryset = []
+    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
```

Comment:
I see what you did here, and it makes sense. Just so we're on the same page, this is how I am understanding this change: `get_hours` is now a different type of function that returns a list of tuples of hours. In other words, it is returning time series data. So our API is now defined as follows:

- `get_events` returns a set of `GEvent`s, for python level set join/intersections
- `query` returns a Django `QuerySet`, for database level join/intersections
- `get_hours` returns time series data in the form of a list of tuples, for consumption by frontend graphs. As per our discussion today, once we generalize this to not just support weeks, I think the most appropriate name would be `get_time_series` and take in parameters something like `calendar_ids`, `start`, `end`, `time_range`.